.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "3.6"
+  # PyPy versions
+  - "pypy3.5"
+  # command to install dependencies
+install:
+  - pip3 install -r requirements.txt
```

Comment:
lets add the line below here to ensure we are able to pypy3.5 to instal all the depeendacies
```yml
 - if [[ $TRAVIS_PYTHON_VERSION == 'pypy3.5' ]]; then pip install cpython; fi
```

Reply:
@brotich what does the `fi` at the end of the line do?

Reply:
this is the Bash syntax for writing an if statement. therefore, it need to be in this syntax instead of the similar call in Python. Does that make sense?

Reply:
Apologies for the typo the package is not named `cpython`, its ``cpython` as in here https://pypi.org/project/Cython/. Kindly update the command above to install this version instead

Reply:
@brotich Makes sense... thanks